meshapp/src/ProfileCardCarousel/Swiper.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useCallback, useEffect } from "react";
+import { useState } from "react";
```

Comment:
Why aren't these combined into `import { React, useCallback, useEffect, useState } from "react";` instead?

Reply:
My bad I forgot to fix that, should be good now